objcToxTests/OCTSubmanagerAvatarsTests.m

```diff
@@ -95,14 +96,38 @@ - (void)testSetAvatarWithNil
 
     OCMExpect([(OCTTox *)self.tox setAvatar:[OCMArg isNil]]);
 
-    [self.subManagerAvatar setAvatar:nil];
+    NSError *error;
+    [self.subManagerAvatar setAvatar:nil error:&error];
 
     //Verify key objects were called
     OCMVerifyAll(self.fileManager);
     OCMVerifyAll(self.tox);
 }
 
-- (void)testPNGData
+- (void)testGetAvatar
+{
+    NSString *path = [kFilePath stringByAppendingPathComponent:kuserAvatarFileName];
+    OCMStub([self.fileManager fileExistsAtPath:path]).andReturn(NO);
```

Comment:
What's the best way to do that without *actually* saving the avatar on the file in the first place?